src/enums/index.ts

```diff
@@ -11,6 +11,7 @@ export * from './filingTypes'
 export * from './itemTypes'
 export * from './partyTypes'
 export * from './modes'
+export * from './name-change-options'
```

Comment:
In the shared repo we use this kebab-case. We also use kebab-case for interfaces. So I'm not starting a new standard but slowly cleaning up something.

Reply:
Eventually start using kebab-case everywhere OR we need to decode on standardization yet?

Reply:
I think we should gradually change enum filenames to kebab-case.

I think we should leave component names in TitleCase.

Thoughts?

Reply:
Yes. This approach makes sense